packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -2,7 +2,10 @@
 :host {
   display: block;
 }
-a[aria-expanded='true'] ~ #info {
+
+/* Add left side blue border if disable border prop disabled or not present */
+:host(:not([disable-border])) a[aria-expanded='true'] ~ #info,
+:host([disable-border='false']) a[aria-expanded='true'] ~ #info {
```

Comment:
I think this is good. Using `:host(:not([disable-border]))` seems like it will keep things easier to read and maintain in the future :+1: 

_(The rest of this is more of a public reminder for myself)_

At some point in the future I'd like to see if we can revisit things like that second line (to remove it), along with other examples like this:

https://github.com/department-of-veterans-affairs/component-library/blob/ba2bca2c9c1251d4960789addba0ad7659ae93f3/packages/web-components/src/components/va-accordion/va-accordion-item.css#L16

The reason is because I don't think that engineers will be using the component like:

```html
<va-additional-info disable-border="false" trigger="Whatever">
  Foo
</va-additional-info>
```

Instead, I think it's more likely that they will just not set `disable-border` if they want to keep the default behavior:

```html
<va-additional-info trigger="Whatever">
  Foo
</va-additional-info>
```

The reason we have our CSS like that is a result of how we are using storybook, so that can be for another ticket.